platforms-whitepaper/v1alpha1/paper.md

```diff
@@ -81,76 +81,77 @@ identity access, infrastructure operations, and app lifecycle.
+necessary to drive their day to day work. For developers for example, this could mean
+having the ability to deploy self-service environments for application development without 
+the overhead of understanding infrastructure components. Or for FinOps, this could
+mean being able to define chargeback rules without understanding getting too in depth of 
+how the metrics are actually collectedfrom cloud-native infrastructure. 
+
+Therefore according to Martin Fowler[[2]], **"a digital platform is a foundation of self-service APIs, tools, services, 
+knowledge and support which are arranged as a compelling internal product."**
+
+It is key to note that a platform is bespoke to an organization, supporting 
+a unique set of users and business needs. Therefore, platforms are not to be confused 
+as just an Internal Developer Portal (IDP) or Service Catalog [[5]]. While those interfaces 
+can certainly be helpful for many organizations, ultimately platform engineering teams
+need to take a platform as product approach and think deeply about the business objectives 
+of their organizations. At its very basic level, a platform could simply be a place where 
+users go for basic actions like user management, technical docs, or incident management if that 
+is what is most needed.
+ 
+However, just like any product, a platform can evolve over time to meet more complex needs of an organization. 
+For example, some example use cases an enterprise can meet with a platform include the following:
 
 1. Developers of products or services can automatically request a complete 
    development environment to support iterative research and development of new 
-   features. This includes spaces in relevant services such as task runners and 
-   artifact storage, membership in designated teams, and publication of connection 
+   features. This includes spaces in relevant services such as Kubernetes clusters, 
+   task runners and artifact storage, membership in designated teams, and publication of connection 
    info such as URLs and secrets.
-1. Developers of products or services can use scenario-specific code and 
-   configuration templates to rapidly bootstrap, develop and deliver new products 
-   and features
-1. Stakeholders in a product or service can observe system and user behavior in 
-   those products and services via instrumentation, dashboards and alerts
-
-A platform is bespoke to an organisation, supporting a unique set of users and 
-business needs. While application developement and delivery is often the first 
-capability a platform supports, it is important to focus on the individual needs of 
-the organization which often surfaces additional opportunities to optimise through 
-a platform offering. Some examples can include data operations, user management, and 
-incident management.
+1. Engineering Teams and SRE Teams have a central location for observability where infrastructure
```

Comment:
Let's agree on a set of use cases together and then write them up to revise this list.

Reply:
addressed based on comments in last meeting - happy to talk through this more